packages/core/src/providers/linkedin.ts

```diff
@@ -78,7 +78,7 @@ export default function LinkedIn<P extends LinkedInProfile>(
     name: "LinkedIn",
     type: "oidc",
     client: { token_endpoint_auth_method: "client_secret_post" },
-    issuer: "https://www.linkedin.com",
+    issuer: "https://www.linkedin.com/oauth",
     jwks_endpoint: "https://www.linkedin.com/oauth/openid/jwks",
```

Comment:
we could drop this line, as it is [discovered](https://www.linkedin.com/oauth/.well-known/openid-configuration)